### PR TITLE
For #1225: Terminate orders and clear wbs on link removal

### PR DESCRIPTION
**Pull request by paulodamaso**
For #1225:
- Added `Orders.remove(order)`
- Added test for `Orders.remove(order)`
- Code for order removal and WBS cleaning on link removal (`terminated_orders_wbs_on_link_removal.groovy`)
- Uncommented tests in `terminate_orders_wbs_on_link_removal` bundle
- Moved bundle to correct directory
- Removed todo.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1494](https://github.com/zerocracy/farm/issues/1494) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1494](https://github.com/zerocracy/farm/issues/1494) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2090311961); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/groovy/com/zerocracy/stk/pm/in/links/terminate_orders_wbs_on_link_removal.groovy [hunk 1]
```diff
@@ -19,13 +19,19 @@ package com.zerocracy.stk.pm.in.links
...
-//  when the link to the Github repo is
-//  removed by the user. After it is implemented uncomment
-//  terminates_orders_wbs_on_link_removal test.
 def exec(Project project, XML xml) {
   new Assume(project, xml).notPmo()
-  new Assume(project, xml).type('Remove link')
+  new Assume(project, xml).type('Project link was removed')
   new Assume(project, xml).roles('PO', 'ARC')
+  Orders orders = new Orders(project).bootstrap()
+  orders.iterate().each{
```
Comment (krzyk) [inline src/main/groovy/com/zerocracy/stk/pm/in/links/terminate_orders_wbs_on_link_removal.groovy]: @paulodamaso  missing space after `each`

File: src/main/java/com/zerocracy/pm/in/Orders.java [hunk 2]
```diff
@@ -168,6 +168,22 @@ public void resign(final String job) throws IOException {
         }
     }
 
+    /**
+     * Remove order.
+     * @param job The order to remove
+     * @throws IOException If fails
+     */
+    public void remove(final String job) throws IOException {
+        try (final Item wbs = this.item()) {
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pm/in/Orders.java]: @paulodamaso this is not a `wbs` item

File: src/test/resources/com/zerocracy/bundles/terminates_orders_wbs_on_link_removal/_after.groovy [hunk 3]
```diff
@@ -14,18 +14,22 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-package com.zerocracy.bundles.terminates_orders_wbs_on_link_removal
+package com.zerocracy.bundles.bundles.terminates_orders_wbs_on_link_removal
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/terminates_orders_wbs_on_link_removal/_after.groovy]: @paulodamaso wrong package name, should be `com.zerocracy.bundles.terminates_orders_wbs_on_link_removal`

File: src/main/groovy/com/zerocracy/stk/pm/in/links/terminate_orders_wbs_on_link_removal.groovy (hunk 1 above)
Comment (paulodamaso) [reply src/main/groovy/com/zerocracy/stk/pm/in/links/terminate_orders_wbs_on_link_removal.groovy]: @krzyk `Qulice` complains when I add this space, so I'm not adding it.

Comment (krzyk) [reply src/main/groovy/com/zerocracy/stk/pm/in/links/terminate_orders_wbs_on_link_removal.groovy]: @paulodamaso So it complains about this line `orders.iterate().each{` and it doesn't about the one below: `wbs.iterate().each {`?

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1494](https://github.com/zerocracy/farm/issues/1494) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by ypshenychka** [comment]
@krzyk thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**